frontend/src/components/models.ts

```diff
@@ -97,7 +97,15 @@ export interface CnsQueryResponse {
 
 // Relayer types
 export type ApiError = { error: string };
-export type TokenListResponse = TokenList | ApiError;
+export interface TokenInfoWithMinSendAmount extends TokenInfo {
```

Comment:
Is there a chance we ever add more data to these types? If so, maybe we rename to `TokenInfoExtended` and `TokenListExtended` or something similar

Reply:
Good call, I agree